lib/static/modules/load-plugin.js

```diff
@@ -105,7 +105,10 @@ async function initPlugin(plugin, pluginName, pluginConfig) {
 // Actual plugin is passed to __hermione_html_reporter_register_plugin__ somewhere in the
 // plugin code
 function executePluginCode(code) {
-    const exec = new Function(`function __hermione_html_reporter_register_plugin__(p) {return p;}; return ${code};`);
+    const getRegisterFn = tool => `function __${tool}_html_reporter_register_plugin__(p) {return p;};`;
+
+    const exec = new Function(`${getRegisterFn('testplane')} ${getRegisterFn('hermione')} return ${code};`);
+
```

Comment:
Add support for testplane html reporter plugins

Reply:
This will work if both are present, right?